main.py

```diff
@@ -1,15 +1,22 @@
-def encrypt(plain, key):
+def encrypt(plain: str, key: int):
     # type: (str, int) -> str
 
     # TODO your code will be here
-    # this is new comment
+    encryptedMsg = ""
+    for char in plain:
+        encryptedMsg += chr((ord(char) - 32 + key) % 95 + 32)
+    return encryptedMsg
     pass
```

Comment:
`pass` is used as a placeholder inside loops, functions, class. So we don't need here.